cdragontoolbox/__main__.py

```diff
@@ -282,6 +283,18 @@ def command_export(parser, args):
         exporter.process(overwrite=overwrite)
 
 
+def command_skn_extract(parser, args):
+    if not os.path.isfile(args.skn):
+        parser.error("SKN file does not exist")
```

Comment:
`does not exist` -> `not found`

Also, you could add the file path: `f"SKN file not found: {args.skn}"`

Reply:
For this I just copied what was done in ``bin_dump``. Should I change the code there as well, to keep it consistent?

Reply:
Yes.